lib/OpenQA/Worker/Jobs.pm

```diff
@@ -163,6 +157,7 @@ sub stop_job($;$) {
     clean_pool();
     $job = undef;
     $worker = undef;
+    exit(0);
```

Comment:
hmm, yeah - useful for debugging. Not for real :)

BTW: it would be great if the worker grabed a job directly after start and not wait for the timer to kick in.